docs/how-to/share-dashboard 2.md

```diff
@@ -3,58 +3,52 @@
 3. Select **Share** to reveal the **Share Panel** and either:  
-   - copy and send the full URL for the dashboard, OR
-   - toggle the **Short URL** option to generate a simple link with a unique identifier
+
+    - copy and send the full URL for the dashboard, OR
+    - toggle the **Short URL** option to generate a simple link with a unique identifier
 
 !!! hint alert alert-success "Tip"
        If your current domain is different than the one specified in the Grafana .INI configuration file, PMM will ask you to correct this mismatch before you can generate a short URL:
     ![!image](../_images/PMM_Common_Panel_Menu_Share.png)
-    To fix this 
-## Share as a PNG file
 
-Rendering images requires the Image Renderer plug-in. If your PMM Admin has not installed this for your PMM instance, you will see the following error message under **Share Panel > Link**.
-![!image](../_images/No_Image_Render_Plugin.png)
+## Share as a PNG file
 
-To install the dependencies:
+Rendering images requires the Image Renderer plug-in running as a separate container. If your PMM Admin has not installed this for your PMM instance, you will see the following error message under **Share Panel > Link**.
 
-1. Connect to your PMM Server Docker container.
+![!image](../_images/No_Image_Render_Plugin.png)
 
-    ```sh
-    docker exec -it pmm-server bash
-    ```
+To enable image rendering:
 
-2. Install Grafana plug-ins.
+1. Deploy the Grafana Image Renderer container alongside PMM Server:
 
     ```sh
-    grafana-cli plugins install grafana-image-renderer
+    docker run -d --name renderer --network=pmm-network grafana/grafana-image-renderer:latest
     ```
 
-3. Restart Grafana.
+2. Stop your existing PMM Server container:
 
-    ```sh
-    supervisorctl restart grafana
+    ```sh  
+    docker stop pmm-server
+    docker rm pmm-server
     ```
 
-4. Install libraries.
+3. Start a new PMM Server container with the required environment variables:
 
     ```sh
-    yum install -y libXcomposite libXdamage libXtst cups libXScrnSaver pango \
-    atk adwaita-cursor-theme adwaita-icon-theme at at-spi2-atk at-spi2-core \
-    cairo-gobject colord-libs dconf desktop-file-utils ed emacs-filesystem \
-    gdk-pixbuf2 glib-networking gnutls gsettings-desktop-schemas \
-    gtk-update-icon-cache gtk3 hicolor-icon-theme jasper-libs json-glib \
-    libappindicator-gtk3 libdbusmenu libdbusmenu-gtk3 libepoxy \
-    liberation-fonts liberation-narrow-fonts liberation-sans-fonts \
-    liberation-serif-fonts libgusb libindicator-gtk3 libmodman libproxy \
-    libsoup libwayland-cursor libwayland-egl libxkbcommon m4 mailx nettle \
-    patch psmisc redhat-lsb-core redhat-lsb-submod-security rest spax time \
-    trousers xdg-utils xkeyboard-config alsa-lib
+    docker run -d \
+    --name pmm-server \
+    --network=pmm-network \
+    -p 8443:443 \
+    -e GF_RENDERING_SERVER_URL=http://renderer:8081/render \
+    -e GF_RENDERING_CALLBACK_URL=https://pmm-server:8443/graph/ \
```

Comment:
```suggestion
    -p 443:443 \
    -e GF_RENDERING_SERVER_URL=http://renderer:8081/render \
    -e GF_RENDERING_CALLBACK_URL=https://pmm-server:443/graph/ \
```

Reply:
I think we need to be consistent with the rest of our docs, where we only mention port 443.